File: tests/fixtures/sample_py/controller.py
from auth import authenticate, refresh_session


class AuthController:
    def login(self, user_id, password):
        return authenticate(user_id, password)

    def refresh(self, session_id):
        return refresh_session(session_id)


def is_admin(user_id):
    return str(user_id).startswith("admin")

File: tests/fixtures/sample_ts/src/controller.ts
import { User, authenticate, refreshSession } from "./auth";

export class AuthController {
  login(userId: string, password: string): boolean {
    return authenticate(userId, password);
  }

  async refresh(sessionId: string): Promise<boolean> {
    return refreshSession(sessionId);
  }
}

export function isAdmin(u: User): boolean {
  return u.id.startsWith("admin");
}
